packages/transpiler/src/statements/select.ts

```diff
@@ -13,18 +13,23 @@ export class SelectTranspiler implements IStatementTranspiler {
     select += node.findFirstExpression(abaplint.Expressions.SQLFieldList)?.concatTokens() + " ";
     select += node.findFirstExpression(abaplint.Expressions.SQLFrom)?.concatTokens() + " ";
 
-    const where = node.findFirstExpression(abaplint.Expressions.SQLCond);
+    let where;
+    for(const sqlCond of node.findAllExpressions(abaplint.Expressions.SQLCond)){
```

Comment:
try this

```suggestion
    for(const sqlCond of node.findDirectExpressions(abaplint.Expressions.SQLCond)){
```

Reply:
"All" will traverse the full tree, "Direct" only direct children

Reply:
I think the SQLCond is not a direct node below the main SELECT node. 
With "findDirectExpressions" i don't find any SQLCond and the tests with WHERE Statements fail:
![image](https://user-images.githubusercontent.com/83953375/170970640-d7c48e49-b7dd-4bce-813f-abb7ab567643.png)

Reply:
it does work, just depends on the context, anyhow, lets just merge it